pkg/linear-pools/contracts/silo-linear-pool/SiloExchangeRateModel.sol

```diff
@@ -33,16 +33,18 @@ contract SiloExchangeRateModel {
      * @dev This function is similar to _accrueInterest function in the Silo's BaseSilo.sol contract
      * which is used to update state data that is necessary
      */
-    function calculateExchangeValue(IShareToken shareToken) external view returns (uint256) {
-        uint256 rcomp = _getCompoundInterestRate(shareToken.silo(), shareToken.asset());
-        ISilo.AssetStorage memory assetStorage = _getAssetStorage(shareToken.silo(), shareToken.asset());
+    function _calculateExchangeValue(IShareToken shareToken) internal view returns (uint256) {
+        ISilo silo = shareToken.silo();
```

Comment:
I think we should actually pass the silo into this function as the first argument, like we do with all the other functions. We are already storing it in both the Pool and the Rebalancer, so we don't need to query it every time we compute a price.